### PR TITLE
Update route.mdx

### PR DESCRIPTION
**Pull request by chironjit**
Added correction to docs referencing access to parameters in routes. It should be context.params.team as opposed to params.team.
 
I've also added typescript example for this case just in case anyone needs it.

### CODE REVIEW

File: docs/02-app/02-api-reference/02-file-conventions/route.mdx [hunk 1]
```diff
@@ -53,12 +53,26 @@ The `request` object is a [NextRequest](/docs/app/api-reference/functions/next-r
...
 ### `context` (optional)
 
-```ts filename="app/dashboard/[team]/route.js"
+```ts filename="app/dashboard/[team]/route.ts" switcher
+type Params = {
+  team: string;
+}
+
+export async function GET(request: Request, context: { params: Params }) {
+  const team = context.params.team // '1'
```
Comment (samcx) [inline docs/02-app/02-api-reference/02-file-conventions/route.mdx:62]:
```suggestion
  const team = params.team // '1'
```
It should be this due to object destructuring.

File: docs/02-app/02-api-reference/02-file-conventions/route.mdx [hunk 2]
```diff
@@ -53,12 +53,26 @@ The `request` object is a [NextRequest](/docs/app/api-reference/functions/next-r
...
+}
+
+// Define params type according to your route parameters (see table below)
+```
+```js filename="app/dashboard/[team]/route.js" switcher
+
 export async function GET(request, context: { params }) {
-  const team = params.team // '1'
+  const team = context.params.team // '1'
```
Comment (samcx) [inline docs/02-app/02-api-reference/02-file-conventions/route.mdx:70]:
```suggestion
  const team = params.team // '1'
```
It should be this due to object destructuring.

**Review by samcx — CHANGES_REQUESTED** [review]

File: docs/02-app/02-api-reference/02-file-conventions/route.mdx (hunk 1 above)
Comment (chironjit) [reply docs/02-app/02-api-reference/02-file-conventions/route.mdx:62]: Not sure if i'm missing something here - but I don't think the you get params destructured here? I had to call context.params.team when i used it or alternatively `const { team } = context.params;`

Comment (samcx) [reply docs/02-app/02-api-reference/02-file-conventions/route.mdx:62]: @chironjit Yes my apologies!

**Comment by ijjk** [comment]
Allow CI Workflow Run

- [ ] approve CI run for commit: f0bd623d8c8f70bf1e695ed6501a569270962c8d

Note: this should only be enabled once the PR is ready to go and can only be enabled by a maintainer

**Comment by ijjk** [comment]
Allow CI Workflow Run

- [ ] approve CI run for commit: f0bd623d8c8f70bf1e695ed6501a569270962c8d

Note: this should only be enabled once the PR is ready to go and can only be enabled by a maintainer

**Review by samcx — APPROVED** [review]

**Comment by samcx** [comment]
@chironjit It looks like there's actually a prettier issue. Can you try formatting the file you've edited?

**Comment by ijjk** [comment]
# Stats from current PR

<details open>
<summary><strong>Default Build</strong></summary>

<br/>

<details>
<summary><strong>General</strong> Overall increase ⚠️</summary>

|  | vercel/next.js canary  | yuvalotem/next.js patch-2 | Change |
| - | - | - | - |
| buildDuration | 13.9s | 13.8s | N/A |
| buildDurationCached | 7.5s | 6.3s | N/A |
| nodeModulesSize | 198 MB | 198 MB | ⚠️ +420 B |
| nextStartRea..uration (ms) | 432ms | 427ms | N/A |

</details>

<details>
<summary><strong>Client Bundles (main, webpack)</strong></summary>

|  | vercel/next.js canary  | yuvalotem/next.js patch-2 | Change |
| - | - | - | - |
| 2453-HASH.js gzip | 30.5 kB | 30.5 kB | N/A |
| 3304.HASH.js gzip | 181 B | 181 B | ✓ |
| 3f784ff6-HASH.js gzip | 53.7 kB | 53.7 kB | N/A |
| 8299-HASH.js gzip | 5.04 kB | 5.04 kB | N/A |
| framework-HASH.js gzip | 45.2 kB | 45.2 kB | ✓ |
| main-app-HASH.js gzip | 242 B | 242 B | ✓ |
| main-HASH.js gzip | 32.2 kB | 32.2 kB | N/A |
| webpack-HASH.js gzip | 1.68 kB | 1.68 kB | N/A |
| Overall change | 45.6 kB | 45.6 kB | ✓ |

</details>

<details>
<summary><strong>Legacy Client Bundles (polyfills)</strong></summary>

|  | vercel/next.js canary  | yuvalotem/next.js patch-2 | Change |
| - | - | - | - |
| polyfills-HASH.js gzip | 31 kB | 31 kB | ✓ |
| Overall change | 31 kB | 31 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages</strong></summary>

|  | vercel/next.js canary  | yuvalotem/next.js patch-2 | Change |
| - | - | - | - |
| _app-HASH.js gzip | 196 B | 197 B | N/A |
| _error-HASH.js gzip | 184 B | 184 B | ✓ |
| amp-HASH.js gzip | 505 B | 505 B | ✓ |
| css-HASH.js gzip | 324 B | 325 B | N/A |
| dynamic-HASH.js gzip | 2.5 kB | 2.5 kB | N/A |
| edge-ssr-HASH.js gzip | 258 B | 258 B | ✓ |
| head-HASH.js gzip | 352 B | 352 B | ✓ |
| hooks-HASH.js gzip | 370 B | 371 B | N/A |
| image-HASH.js gzip | 4.21 kB | 4.21 kB | ✓ |
| index-HASH.js gzip | 259 B | 259 B | ✓ |
| link-HASH.js gzip | 2.67 kB | 2.67 kB | N/A |
| routerDirect..HASH.js gzip | 314 B | 312 B | N/A |
| script-HASH.js gzip | 386 B | 386 B | ✓ |
| withRouter-HASH.js gzip | 309 B | 309 B | ✓ |
| 1afbb74e6ecf..834.css gzip | 106 B | 106 B | ✓ |
| Overall change | 6.57 kB | 6.57 kB | ✓ |

</details>

<details>
<summary><strong>Client Build Manifests</strong></summary>

|  | vercel/next.js canary  | yuvalotem/next.js patch-2 | Change |
| - | - | - | - |
| _buildManifest.js gzip | 481 B | 484 B | N/A |
| Overall change | 0 B | 0 B | ✓ |

</details>

<details>
<summary><strong>Rendered Page Sizes</strong></summary>

|  | vercel/next.js canary  | yuvalotem/next.js patch-2 | Change |
| - | - | - | - |
| index.html gzip | 530 B | 529 B | N/A |
| link.html gzip | 541 B | 541 B | ✓ |
| withRouter.html gzip | 524 B | 523 B | N/A |
| Overall change | 541 B | 541 B | ✓ |

</details>

<details>
<summary><strong>Edge SSR bundle Size</strong></summary>

|  | vercel/next.js canary  | yuvalotem/next.js patch-2 | Change |
| - | - | - | - |
| edge-ssr.js gzip | 95.1 kB | 95.1 kB | N/A |
| page.js gzip | 3.06 kB | 3.07 kB | N/A |
| Overall change | 0 B | 0 B | ✓ |

</details>

<details>
<summary><strong>Middleware size</strong></summary>

|  | vercel/next.js canary  | yuvalotem/next.js patch-2 | Change |
| - | - | - | - |
| middleware-b..fest.js gzip | 626 B | 623 B | N/A |
| middleware-r..fest.js gzip | 151 B | 151 B | ✓ |
| middleware.js gzip | 25.5 kB | 25.5 kB | N/A |
| edge-runtime..pack.js gzip | 839 B | 839 B | ✓ |
| Overall change | 990 B | 990 B | ✓ |

</details>

<details>
<summary><strong>Next Runtimes</strong></summary>

|  | vercel/next.js canary  | yuvalotem/next.js patch-2 | Change |
| - | - | - | - |
| app-page-exp...dev.js gzip | 171 kB | 171 kB | ✓ |
| app-page-exp..prod.js gzip | 96.5 kB | 96.5 kB | ✓ |
| app-page-tur..prod.js gzip | 98.3 kB | 98.3 kB | ✓ |
| app-page-tur..prod.js gzip | 92.7 kB | 92.7 kB | ✓ |
| app-page.run...dev.js gzip | 149 kB | 149 kB | ✓ |
| app-page.run..prod.js gzip | 91.2 kB | 91.2 kB | ✓ |
| app-route-ex...dev.js gzip | 21.3 kB | 21.3 kB | ✓ |
| app-route-ex..prod.js gzip | 15 kB | 15 kB | ✓ |
| app-route-tu..prod.js gzip | 15 kB | 15 kB | ✓ |
| app-route-tu..prod.js gzip | 14.8 kB | 14.8 kB | ✓ |
| app-route.ru...dev.js gzip | 21 kB | 21 kB | ✓ |
| app-route.ru..prod.js gzip | 14.8 kB | 14.8 kB | ✓ |
| pages-api-tu..prod.js gzip | 9.52 kB | 9.52 kB | ✓ |
| pages-api.ru...dev.js gzip | 9.8 kB | 9.8 kB | ✓ |
| pages-api.ru..prod.js gzip | 9.52 kB | 9.52 kB | ✓ |
| pages-turbo...prod.js gzip | 22.3 kB | 22.3 kB | ✓ |
| pages.runtim...dev.js gzip | 22.9 kB | 22.9 kB | ✓ |
| pages.runtim..prod.js gzip | 22.3 kB | 22.3 kB | ✓ |
| server.runti..prod.js gzip | 50.5 kB | 50.5 kB | N/A |
| Overall change | 897 kB | 897 kB | ✓ |

</details>

<details>
<summary><strong>build cache</strong></summary>

|  | vercel/next.js canary  | yuvalotem/next.js patch-2 | Change |
| - | - | - | - |
| 0.pack gzip | 1.56 MB | 1.56 MB | N/A |
| index.pack gzip | 106 kB | 105 kB | N/A |
| Overall change | 0 B | 0 B | ✓ |

</details>

<details>
<summary><strong>Diff details</strong></summary>

<details>
<summary>Diff for <strong>middleware.js</strong></summary>

Diff too large to display
</details>
<details>
<summary>Diff for <strong>edge-ssr.js</strong></summary>

Diff too large to display
</details>
<details>
<summary>Diff for <strong>server.runtime.prod.js</strong></summary>

Diff too large to display
</details>

</details>

</details>
Commit: 01b390d6f5996acee86101c042afcc36a0a8f4ad

**Pull request merged**